recipes/astropy-helpers/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "astropy-helpers" %}
+{% set version = "1.3.1" %}
+{% set sha256 = "7640c5582e518a3ed2f2b5a209d188f1d5099ab33cc1e861a2c123e0e80c9fcd" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed=files.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  requires:
+    - pytest
+  imports:
+    - astropy_helpers
+  commands:
+    - py.test --pyargs astropy_helpers.tests
+    - py.test --pyargs astropy_helpers.tests.test_ah_bootstrap
+    - py.test --pyargs astropy_helpers.tests.test_git_helpers
+    - py.test --pyargs astropy_helpers.tests.test_setup_helpers
+    - py.test --pyargs astropy_helpers.tests.test_utils
+
+about:
+  home: http://www.astropy.org/
+  license: BSD
+  summary: Helpers for the "astropy" project.
+  dev_url: https://github.com/astropy/astropy-helpers
+
+extra:
+  recipe-maintainers:
+    - pkgw
```

Comment:
At a minimum this should also include those pinged in the comment by @jakirkham, and @bsipocz  and @pllim